wremnants/HDF5Writer.py

```diff
@@ -253,6 +256,12 @@ def write(self,
 
                 # nominal histograms of prediction
                 norm_proc_hist = dg.groups[proc].hists[chanInfo.nominalName]
+                if "helicity" in chanInfo.nominalName:
```

Comment:
rather than hardcoding the behaviour based on the channel name it would probably be better to have a configurable option in the card tool

then these if statements could become ```if chanInfo.exponentialTransform``` or whatever

Reply:
ok this can be taken care of afterwards in any case